src/index.html

```diff
@@ -4,9 +4,383 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="index.html">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <img
+          class="card__good"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        >
+
+        <h2 class="card__description">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="price">Price:</span>
+          <span class="money">$2,199</span>
+        </div>
```

Comment:
these elements should be as elements of card block
```suggestion
          <p class="card__review">Reviews: 5</p>
        </div>

        <div class="card__price">
          <span class="card__price-text">Price:</span>
          <span class="card__price-value">$2,199</span>
        </div>
```